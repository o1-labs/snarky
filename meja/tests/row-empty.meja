let f = fun (x : [< `A], y : [< `B], b) => {
  if (b) { x; } else { y; };
};
