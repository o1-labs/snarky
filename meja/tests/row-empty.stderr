File "tests/row-empty.meja", line 2, characters 23-24:
Error: Incompatible types ([<`A] as 'a) and ([<`B] as 'b):
       Cannot unify ([<`A] as 'a) and ([<`B] as 'b):
       the resulting row would be empty.
